app/controllers/users_controller.rb

```diff
@@ -29,8 +29,9 @@ def edit
 
   def update
     if user_params[:password] === ''
+      @user.update(user_params)
       flash[:error] = "Password is a mandatory field"
-      redirect_to edit_user_path(@user)
```

Comment:
This does not work. It breaks the referer setup for the account deletion feature. Unfortunately we have to keep the redirect otherwise the authorization check for getting to the delete-account-page breaks. Maybe we can find a work around?